package.json

```diff
@@ -23,6 +23,7 @@
   "nyc": {
     "exclude": [
       "**/*ThreeExtended",
+      "**/*src/Utils/placeObjectOnGround.js",
```

Comment:
Why would you exclude it ?

Reply:
Because I deprecated `placeObjectOnGround` until I will test it or remove it.

Reply:
Ok